### DISCUSSION TITLE
Arrays of features as input in Tabular predictor

### DISCUSSION THREAD

**Discussion by anthipapa** [autogluon/autogluon#3109] [Q&A]
Is there a way to use np.arrays or lists of features as input?
For example if a training  dataframe looks like this
```

       numeric_types                                        probability  label
0                  8  [-1.9030330674777605, -5.65052815657148, -4.61...      1
1                  2  [-3.909052222527045, -4.2751743584127215, -4.1...      0
2                  6                             [-0.06063525518052539]      0
3                  1  [-2.6584356879041864, -3.393729505525358]              1
```

and I try
```
from autogluon.tabular import TabularPredictor

label = 'label'
predictor = autogluon.tabular.TabularPredictor(label=label, eval_metric = 'f1', verbosity = 4)
predictor.fit(train)
```

I get `TypeError: unhashable type: 'list'`

The numbers inside the list are log probabilities to be classified (binary classification task), and need to be in this format and not aggregated (e.g. summing them) or split. The 'types' field refers to a category these probabilities belong to (out of 7). Is there a solution of workaround for this type of problem?

**Comment by gradientsky** [comment]
Unfortunately, the variable sequences classification is not supported out-of-the-box.
